### PR TITLE
write name David Forgione in Team.md

### PR DESCRIPTION
**Pull request by irdof**

### CODE REVIEW

File: Team.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ E. K.
 Favre Lenny
 F. A.
 Fluck Yoann
-F. D.
+FORGIONE. DAVID.
```
Comment (metc) [inline Team.md:11]: Do not use capital letters, please :)

**Review by metc — CHANGES_REQUESTED** [review]

File: Team.md (hunk 1 above)
Comment (metc) [reply Team.md:11]: Points should be removed as well.

**Review by metc — CHANGES_REQUESTED** [review]
Thanks for your contribution.
Please rebase (`git rebase`) your work on top of the latest `master` branch so I can merge your work. Thank you.

File: Team.md (hunk 1 above)
Comment (irdof) [reply Team.md]: Désolé je n'ai pas vu les messages, je le fais maintenant!

**Pull request closed without merging**